database/migrations/2023_12_12_171400_alert_rule_note.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::table('alert_rules', function (Blueprint $table) {
+            $table->text('notes');
```

Comment:
I would either allow "null" as a value (and adapt the code for it all along) or define a default value of `""` (empty string).